templates/telco/osm/onboarding-network-functions.html

```diff
@@ -0,0 +1,191 @@
+{% extends "templates/one-column.html" %}
+
+{% block title %}Onboarding Network Functions{% endblock %}
+
+{% block meta_description %}Onboarding network functions with Canonical Open Source MANO.{% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1_35nO3riTFlSFSp2B7FFdl60cjZe7Owwv0bcjY1s8Kg/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped is-deep is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>Onboarding Network Functions</h1>
+      <p><a class="p-link--external" href="https://osm.etsi.org/" alt="">Open Source MANO</a> provides a platform to easily migrate your traditional telco functions to virtualized and cloud-native environments. It supports onboarding of all types of network functions:</p>
+      <ul>
+          <li>Virtualized Network Functions (VNF)</li>
+          <li>Cloud-native Network Functions (CNF)</li>
+          <li>Physical Network Functions (PNF)</li>
+      </ul>
+      <p>
+        <a class="p-button--positive" href="#get-in-touch">
```

Comment:
This href should link to a contact-us page with the class `js-invoke-modal` to bring up the modal

Reply:
In the copy doc this is linking to "Tell us about your network function" and it has been confirmed that's how they want it.